source/markdown/data_controle_2024.Rmd

```diff
@@ -0,0 +1,1572 @@
+
+```{r}
+soorten_wet_mas <- read_csv2(file.path("data", "SOVON", "soorten_wet_mas.csv"))
+
+test_wet_names <- manual_taxon_data %>%
+  select("dwc_taxonID", "dwc_vernacularName", "scientificName",
+         "authorship") %>%
+  mutate(authorship = trimws(authorship)) %>%
+  rowwise() %>%
+  mutate(scientificName = gsub(authorship, "", scientificName),
+         dwc_scientificName = trimws(gsub("\\(.*?\\)", "", scientificName))
+  ) %>%
+  select(-"authorship", -"scientificName") %>%
+  left_join(soorten_wet_mas, by = join_by("dwc_taxonID" == "euring")) %>%
+  select(-"groepnr")
+```
+
+De volgende soorten zijn niet in de lijst van sovon aanwezig.
+`dwc_taxonID` is de id voor de Nederlandstalige soortnaam van Sovon, `naam` is de Nederlandstalige naam op de lijst van Sovon en `wetenschap` is de wetenschappelijke naam op de lijst van Sovon.
+
+```{r}
+missing_soorten_wet_mas <- test_wet_names %>%
+  filter(is.na(naam))
+
+missing_soorten_wet_mas %>%
+  kable()
+```
+
+Hoe zit het met de wetenschappelijke namen van onze functie (`dwc_scientificName`) en van de lijst van Sovon. Komen deze overeen?
+
+```{r}
+mismatches_wet_names <- test_wet_names %>%
+  filter(!is.na(naam)) %>%
+  mutate(test_wet = dwc_scientificName == wetenschap)
+
+mismatches_wet_names %>%
+  filter(!test_wet) %>%
+  select(-"test_wet") %>%
+  kable()
+```
+
+Er zijn een aantal mismatches:
+
+1. Species versus subspecies
+  - Vormen in principe geen probleem
+2. Andere genera
+  - Braamsluiper: *Curruca curruca* is "Synonym of Sylvia curruca (Linnaeus, 1758)" volgens GBIF backbone taxonomy
+  - Fluiter: met 2 ll'en volgens GBIF backbone taxonomy(op wikipedia ook met 1 l ...)
+  - Grasmus: *Curruca communis* is "Homotypic synonym of Sylvia communis Latham, 1787" volgens GBIF backbone taxonomy
+  - Matkop: zowel *Parus montanus* als *Poecile montanus* zijn accepted in GBIF backbone taxonomy
+
+Ziet er over het algemeen ok uit.
+
+## Finale dataset
+
+Na DwC mapping bekomen we de finale dataset voor publicatie.
+We bekijken hier of er nog tekortkomingen zijn.
+
+```{r}
+darwincore_mapping <- tar_read("darwincore_mapping", store = targets_store)
+dwc_mapping_final <- tar_read("dwc_mapping_final", store = targets_store)
+```
+
+Welke kolommen zitten niet in de dataset?
+
+```{r}
+sort(colnames(darwincore_mapping)[
+    grepl("^raw\\_", colnames(darwincore_mapping))
+  ])
+```
+
+- Locaties telpunten ook in dataset (`raw_x_coord`, `raw_y_coord`, `raw_crs`) --> Event versus occurrence dataset opsplitsen?
+- Afstand tot telpunt: `raw_distance2plot` --> welke term?
+- Zijn variabelen relevant om mee te geven?
+  - `raw_area_prop_sb` --> Overlap steekproefkader zenodo?
+  - `raw_sbp` --> Overlap steekproefkader zenodo?
+  - `raw_openheid_klasse` --> Overlap steekproefkader zenodo?
+  - `raw_regio` --> Overlap steekproefkader zenodo?
+  - `raw_periode_in_jaar`: 1 van 4 telperiodes
+  - `raw_status_teller`: is de waarnemer een professional of een vrijwilliger?
```

Comment:
Not really necessary for GBIF in my opinion

Reply:
added comments to file in this commit: [790694b](https://github.com/inbo/mbag-mas/pull/37/commits/790694b551ecd34d0ce7d05c71e55859498a8df3)